frontend/components/panel/js/panel.service.ts

```diff
@@ -495,6 +520,22 @@ export class PanelService {
 
 	}
 
+	/**
+	 * Download server response JSON file from panel menu
+	 *
+	 * @param content The JSON response to download
+	 * @param fileName Choice of filename : "risks.json", "issues.json", "groups.json"
+	 * @param contentType content type for downloaded file : "text/plain", "application/json"
+	 */
+
+	public downloadJSON(content, fileName, contentType) {
+		const a = document.createElement("a");
```

Comment:
Instead of creating a dom element and emulate a click, can we do it with `this.$window.open(blobURL, "_blank");`?

Genuine question, I don't know if it works.

Reply:
Hey Carmen , I couldn't seem replicate the download functionality using just window.open. It would only take me to the url with a page of JSON and not download it. Let me know your thoughts :)